zulip/zulip/__init__.py

```diff
@@ -1320,6 +1320,14 @@ def reactivate_user_by_id(self, user_id: int) -> Dict[str, Any]:
             method="POST",
         )
 
+    def update_user_by_id_compatible(self, user_id: int, **request: Any) -> Dict[str, Any]:
```

Comment:
I think it's worth having functions like this come with a block comment explaining the API change.  Potentially just copy-pasting the relevant section of the API documentation would be appropriate.

Reply:
I think in any cases, we will need to have comments explaining why we will do a feature_level comparison. Otherwise the feature level just looks like a magic number.